### DISCUSSION TITLE
Downloading artist images?

### DISCUSSION THREAD

**Discussion by Bigmack3000** [beetbox/beets#5368] [Q&A]
Hey, i have a library that is already tagged and organized, but I'm looking for something that can download artist images into each artist's folder.  Is this something beets can do?

**Comment by wisp3rwind** [comment]
No, it's a long-standing feature request, see for example https://github.com/beetbox/beets/issues/889

**Reply by Bigmack3000** [reply]
Thanks. Do you know of a service that can do this?
